client/my-sites/site-settings/date-time-format/date-time-format-options.jsx

```diff
@@ -0,0 +1,272 @@
+							disabled={ isRequestingSettings }
+							name="date_format"
+							onChange={ handleRadio }
+							value={ format }
+						/>
+						<span>{ today.format( phpToMomentDatetimeFormat( format ) ) }</span>
+					</FormLabel>
+				) }
+				<FormLabel className={ customFieldClasses }>
+					<FormRadio
+						checked={ isCustomFormat }
+						disabled={ isRequestingSettings }
+						name="date_format"
+						onChange={ handleRadio }
+						value={ date_format }
+					/>
+					<span>
+						{ translate( 'Custom' ) }
+						<FormInput
+							disabled={ isRequestingSettings }
+							name="date_format_custom"
+							onChange={ onChangeField( 'date_format' ) }
+							type="text"
+							value={ date_format || '' }
+						/>
+						<span className="date-time-format__custom-preview">
+							{ isCustomFormat && date_format
+								? today.format( phpToMomentDatetimeFormat( date_format ) )
+								: ''
+							}
+						</span>
+				</span>
+				</FormLabel>
+			</FormFieldset>
+		);
+	}
+
+	timeFormatOption() {
+		const {
+			fields: { time_format, timezone_string },
+			handleRadio,
+			isRequestingSettings,
+			moment,
+			onChangeField,
+			translate,
+		} = this.props;
+
+		const defaultFormats = [ 'g:i a', 'g:i A', 'H:i' ];
+		const isCustomFormat = ! includes( defaultFormats, time_format );
+		const today = startsWith( timezone_string, 'UTC' )
+			? moment().utcOffset( timezone_string.substring( 3 ) * 60 )
+			: moment.tz( timezone_string );
+
+		const customFieldClasses = classNames(
+			'date-time-format__custom-field',
+			{ 'is-custom': isCustomFormat }
+		);
+
+		return (
+			<FormFieldset>
+				<FormLabel>
+					{ translate( 'Time Format' ) }
+				</FormLabel>
+				{ defaultFormats.map( ( format, key ) =>
+					<FormLabel key={ key }>
+						<FormRadio
+							checked={ format === time_format }
+							disabled={ isRequestingSettings }
+							name="time_format"
+							onChange={ handleRadio }
+							value={ format }
+						/>
+						<span>{ today.format( phpToMomentDatetimeFormat( format ) ) }</span>
+					</FormLabel>
+				) }
+				<FormLabel className={ customFieldClasses }>
+					<FormRadio
+						checked={ isCustomFormat }
+						disabled={ isRequestingSettings }
+						name="time_format"
+						onChange={ handleRadio }
+						value={ time_format }
+					/>
+					<span>
+						{ translate( 'Custom' ) }
+						<FormInput
+							disabled={ isRequestingSettings }
+							name="time_format_custom"
+							onChange={ onChangeField( 'time_format' ) }
+							type="text"
+							value={ time_format || '' }
+						/>
+						<span className="date-time-format__custom-preview">
+							{ isCustomFormat && time_format
+								? today.format( phpToMomentDatetimeFormat( time_format ) )
+								: ''
+							}
+						</span>
+					</span>
+					<FormSettingExplanation>
+						<ExternalLink href="https://codex.wordpress.org/Formatting_Date_and_Time" icon>
+							{ translate( 'Learn more about date and time formatting.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Documentation on date and time formatting.' )` *ES Score: 6.55*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).